### ISSUE TITLE
Revise English documents 

### ISSUE THREAD

**Issue by Technoboy-** [apache/shardingsphere-elasticjob#1441]
In the beginning, our contributors translate the documents in a little hurry that may not be accurate. If some computer professional experts with fluent English or English major can help revise and correct.

**Comment by sunilgknair051** [comment]
Hi, I Would love to get assigned. But please specify the files. Or is it only the readme.md?

**Comment by Technoboy-** [comment]
> Hi, I Would love to get assigned. But please specify the files. Or is it only the readme.md?

Hi, this issue means the whole en docs. You can try to revise from chapter to chapter.

**Comment by sunilgknair051** [comment]
@Technoboy- , I still am a bit confused. Can you please specifically tell me which files need changes?
shardingsphere-elasticjob/docs/  - this right?   If yes, there are characters in them I need to translate as well?
or did you mean dev-manual or the user-manual??
need you to link me to the folders that I need to work on.
And please assign me as well.
Thanks!

**Comment by Technoboy-** [comment]
> @Technoboy- , I still am a bit confused. Can you please specifically tell me which files need changes?
> shardingsphere-elasticjob/docs/ - this right? If yes, there are characters in them I need to translate as well?
> or did you mean dev-manual or the user-manual??
> need you to link me to the folders that I need to work on.
> And please assign me as well.
> Thanks!

Yes, under the folder of docs/content. all the subfolders that contained en.md suffixes are preferred to revise. 
For some sentences or paragraphs maybe not translate right. Please help to check. 
Thanks.
